nat-lab/Pipfile

```diff
@@ -51,7 +51,6 @@ asyncssh = "==2.14.2"
 aiodocker = "*"
 scapy="*"
 paho-mqtt = "*"
-types-paho-mqtt = "*"
```

Comment:
why types are not required anymore?

Reply:
paho-mqtt have types included into main package since version 2. types-paho-mqtt should only be used with version prior to that and MUST NOT be used with v2 because it is confusing type checkers